distribution/images/multi-db/init-database.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+set -eu
+
+# Wait for PostgreSQL to start
+until pg_isready -U "root"; do
+  echo "Waiting for PostgreSQL to start..."
+  sleep 2
+done
+
+# Create the database and users dynamically
+echo "Creating databases and users..."
+
+psql -v ON_ERROR_STOP=1 --username root <<-EOSQL
+  CREATE USER crux_user WITH ENCRYPTED PASSWORD '${CRUX_POSTGRES_PASSWORD}';
```

Comment:
This way if you change the env and define it in the docker-compose it will be ignored. It has a default that's good, but env variables should be respected by the script as well, both user dbname and password.